projects/plugins/jetpack/sal/class.json-api-site-base.php

```diff
@@ -1458,4 +1458,26 @@ public function get_launchpad_checklist_tasks_statuses() {
 
 		return array();
 	}
+
+	/**
+	 * Get the site option for the connected GitHub repo name.
+	 */
+	public function get_github_hosting_repo_name() {
+		return get_option( 'github_hosting_repo_name', '' );
+	}
+
+	/**
+	 * Get the site option for the connected GitHub branch name.
+	 */
+	public function get_github_hosting_branch_name() {
+		return get_option( 'github_hosting_branch_name', '' );
+	}
+
+	/**
+	 * Get the site option for the base path to use when copying
+	 * files from the connected GitHub repo to the Atomic file system.
+	 */
+	public function get_github_hosting_base_path() {
+		return get_option( 'github_hosting_base_path', '' );
```

Comment:
Is it too late to change those option names, and prefix them with `wpcom_` or `jetpack_`? I worry that such option names will be used by other plugins in the WordPress.org directory, and would conflict with our implementation in the future.

Reply:
Let's prefix them with `wpcom_` 👍

Reply:
I've updated the PR to use the `wpcom` prefix.